ignite-base/.flowconfig

```diff
@@ -55,4 +64,4 @@ suppress_comment=\\(.\\|\n\\)*\\$FlowFixedInNextDeploy
 unsafe.enable_getters_and_setters=true
 
 [version]
-^0.32.0
+^0.33.0
```

Comment:
This errors for me: I'm seeing this issue.

https://github.com/facebook/flow/issues/2548